components/_cover-photo.scss

```diff
@@ -72,7 +78,8 @@
 
 .cover-photo-cta {
   margin-top: 1.0em;
-  @include padding(1.75em 1.5em);
-  width: 80%;
-  font-weight: 600;
+  width: 50%;
+  @include media($mobile-only) {
```

Comment:
These styles are all designed mobile first. You should be specifying how they scale up, not scale down.